lib/UnoCore/Targets/Android/Android.uxl

```diff
@@ -219,6 +219,38 @@
 
     <!-- Icons -->
 
+    #if @(Project.Android.Icons.AdaptiveIconEnabled:Test(1, 0))
```

Comment:
`#if..#else..#endif` are not supported in UXL. I suggest removing those and the duplicated `<ImageFile>` elements (that are inside the `#else..#endif` block).

Reply:
cool, yeah